src/tools/rest-frontend/resources/assets/js/controllers/main/MainController.js

```diff
@@ -12,7 +12,7 @@ module.exports = function ($rootScope, $scope, Logger, $state, $anchorScroll, we
 
     // build date
     $scope.builddate = {
-        pretty: (new Date(document.querySelector('meta[name="build-date"]').getAttribute('content'))).toLocaleString()
+        pretty: (new Date(document.querySelector('meta[name="build-date"]').getAttribute('content'))).toUTCString()
```

Comment:
That info is not available as far as I know, but we could use [Date.toString()](http://www.w3schools.com/jsref/jsref_tostring_date.asp) instead (I'm afraid that this takes quite a lot of space though).